services/intersection-api/api/src/main/java/us/dot/its/jpo/ode/api/models/AggregationResult.java

```diff
@@ -0,0 +1,24 @@
+package us.dot.its.jpo.ode.api.models;
+
+import java.util.List;
+
+public class AggregationResult<T> {
+    private List<T> data;
```

Comment:
suggestion(non-blocking): `data` is an overloaded, and therefore a somewhat meaningless name (to me). It could be more descriptive if we named this field. The plural indicates it's a collection while `data` doesn't immediately tell me if I'm working with a single object or a collection.
```suggestion
    private List<T> results;
```

Reply:
updated